sos/report/plugins/virsh.py

```diff
@@ -82,4 +82,20 @@ def setup(self):
             for n in nodedev_output['output'].splitlines():
                 self.add_cmd_output(
                     '{0} nodedev-dumpxml {1}'.format(cmd, n), foreground=True)
+
+    def postproc(self):
+        match_exp = r"(\s*passwd\s*=\s*\")([^\"]*)(\".*\s*)"
```

Comment:
Why the trailing `\s*`, which should be contained in `.*` already?

Otherwise ACK, thanks for the PR.

Reply:
Haha! Yeah, that really looks redundant :-).